src/BsRecharts__LabelList.re

```diff
@@ -0,0 +1,41 @@
+// http://recharts.org/en-US/api/LabelList
+
+[@bs.module "recharts"] [@react.component]
+external make:
+  (
+    ~className: string=?,
+    ~clockwise: string=?,
+    ~content: 'content=?,
+    ~dataKey: string=?,
+    ~fill: string=?,
+    ~formatter: 'formatter=?,
+    ~id: string=?,
+    ~labelFormatter: 'labelFormatter=?,
+    ~valueAccessor: 'valueAccessor=?,
```

Comment:
@liubko, I see that we allow some untyped props. Is there any reason not to type them?

Reply:
In many cases, props are polymorphic, and it's not straightforward to type, but ofc it doesn't mean we shouldn't.

Reply:
also if it's too cumbersome to annotate the argument we can at least do it partially, like in case with `valueAccessor` we know that this has to be a function

```
~valueAccessor: ('dataEntry, int) => `value,
```
https://github.com/recharts/recharts/blob/7f913f27f35c688e3e3554363c1bf8899d58ce06/src/component/LabelList.tsx#L51